tools/accuracy_checker/dataset_definitions.yml

```diff
@@ -368,6 +368,39 @@ datasets:
     annotation: crossroad-0078.pickle
     dataset_meta: crossroad-0078.json
 
+  - name: crossroad_extra_yg
+    data_source: crossroad_extra_yg/val_data
+    annotation_conversion:
+      converter: mscoco_detection
+      annotation_file: crossroad_extra_yg/val_coco_no_bg.json
+      has_background: True
+      sort_annotations: True
+      use_full_label_map: True
+    annotation: crossroad_extra_yg.pickle
+    dataset_meta: crossroad_extra_yg.json
+
+  - name: crossroad_extra_yg_person
+    data_source: crossroad_extra_yg/val_data
+    annotation_conversion:
+      converter: mscoco_detection
+      annotation_file: crossroad_extra_yg/val_coco_no_bg_person.json
+      has_background: True
+      sort_annotations: True
+      use_full_label_map: True
+    annotation: crossroad_extra_yg_person.pickle
+    dataset_meta: crossroad_extra_yg_person.json
+
+  - name: crossroad_extra_yg_vehicle
```

Comment:
not sure that it is good idea to name datasets in public repo in YG honor :) or you mean something another?

Reply:
I thought no one would notice or understand. Ok, I'll rename it